src/plugins/fileViewer/fileViewer.css

```
@@ -0,0 +1,29 @@
+.file-viewer.container {
```

classnames should be prefixed with `vc-` to differentiate them from discord classes and avoid conflicts

```suggestion
.vc-fileViewer.container {
```